packages/algob/test/lib/tx.ts

```diff
@@ -22,3 +33,89 @@ describe("Note in TxParams", () => {
     assert.deepEqual(result, encoder.encode(noteb64), "noteb64 not encoded");
   });
 });
+
+describe("Opt-In to ASA", () => {
+  function mkASA (): types.ASADef {
+    return {
+      total: 1,
+      decimals: 1,
+      unitName: 'ASA',
+      defaultFrozen: false
+    };
+  }
+
+  const s: SuggestedParams = {
+    flatFee: false,
+    fee: 100,
+    firstRound: 2,
+    lastRound: 100,
+    genesisID: 'testnet-v1.0',
+    genesisHash: 'SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI='
+  };
+
+  const account = {
+    addr: 'UDF7DS5QXECBUEDF3GZVHHLXDRJOVTGR7EORYGDBPJ2FNB5D5T636QMWZY',
+    sk: new Uint8Array([28, 45, 45, 15, 70, 188, 57, 228, 18, 21, 42,
+      228, 33, 187, 222, 162, 89, 15, 22, 52, 143, 171, 182, 17, 168,
+      238, 96, 177, 12, 163, 243, 231, 160, 203, 241, 203, 176, 185,
+      4, 26, 16, 101, 217, 179, 83, 157, 119, 28, 82, 234, 204,
+      209, 249, 29, 28, 24, 97, 122, 116, 86, 135, 163, 236, 253])
+  };
```

Comment:
don't we have a common account we use for tests? If not, let's create a test package with test data (we can put SuggestedParams there as well).

Reply:
ok

Reply:
done created tx file mocks folder.